base/essentials.jl

```diff
@@ -147,7 +147,7 @@ macro specialize(vars...)
 end
 
 """
-    @isdefined s -> Bool
+    @isdefined s::Bool
```

Comment:
This parses as calling the macro on `:(s::Bool)`, which isn't great.

Reply:
Yep! I intended to skip macros in this PR but accidentally included that. Should be fixed (probably just reverted/left alone)